h/static/scripts/controllers.coffee

```diff
@@ -170,45 +198,43 @@ class ViewerController
     loaded = []
 
     loadAnnotations = ->
-      if annotator.tool is 'highlight'
+      if annotationUI.tool is 'highlight'
         return unless auth.user
         query = user: auth.user
 
-      for p in annotator.providers
+      for p in crossframe.providers
         for e in p.entities when e not in loaded
           loaded.push e
-          store.SearchResource.get angular.extend(uri: e, query), (results) ->
-            annotator.loadAnnotations(results.rows)
+          r = store.SearchResource.get angular.extend(uri: e, query), (results) ->
```

Comment:
I agree, although this PR is already huge and I'd like to avoid adding any more unrelated changes than strictly necessary.